__tests__/app.test.js

```diff
@@ -145,6 +145,33 @@ describe('/api/articles', () => {
             expect(response.body.msg).toBe('Bad request');
         });
     })
+    test('GET:200 accept a valid topic query, which has articles ', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toHaveLength(12)
+            body.articles.forEach((article) => {
+                expect(article.topic).toBe('mitch')              
+            })
```

Comment:
Good test here, checking length and the only thing you need to check is the topic, no need to check all the properties again